entproto/cmd/protoc-gen-entgrpc/cast.go

```diff
@@ -51,3 +52,27 @@ func (g *serviceGenerator) castToProtoFunc(fld *entproto.FieldMappingDescriptor)
 		return nil, fmt.Errorf("entproto: no mapping for pb field type %q", pbd.GetType())
 	}
 }
+
+func (g *serviceGenerator) castToEntFunc(fd *entproto.FieldMappingDescriptor) (interface{}, error) {
+	var fld *gen.Field
+	if fd.IsEdgeField {
+		fld = fd.EntEdge.Type.ID
+	} else {
+		fld = fd.EntField
+	}
```

Comment:
Not mandatory, but I usaully do:

```suggestion
	fld := fd.EntField
	if fd.IsEdgeField {
		fld = fd.EntEdge.Type.ID
	}
```